### PR TITLE
Implement broadcast for some services

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,17 @@
 | master | [![Build master](https://ci.camptocamp.com/buildStatus/icon?job=geospatial/c2cwsgiutils/master)](https://ci.camptocamp.com/job/geospatial/job/c2cwsgiutils/job/master/) | [![Requirements master](https://requires.io/github/camptocamp/c2cwsgiutils/requirements.svg?branch=master)](https://requires.io/github/camptocamp/c2cwsgiutils/requirements/?branch=master)
 | release_0 | [![Build release_0](https://ci.camptocamp.com/buildStatus/icon?job=geospatial/c2cwsgiutils/release_0)](https://ci.camptocamp.com/job/geospatial/job/c2cwsgiutils/job/release_0/) | [![Requirements release_0](https://requires.io/github/camptocamp/c2cwsgiutils/requirements.svg?branch=release_0)](https://requires.io/github/camptocamp/c2cwsgiutils/requirements/?branch=release_0) |
 
-This is a python 3 library providing common tools for Camptocamp WSGI
+This is a python 3 library (>=3.5) providing common tools for Camptocamp WSGI
```
Comment (sbrunner) [inline README.md]: s/python/Python/

File: README.md [hunk 2]
```diff
@@ -43,6 +40,9 @@ With pip:
 pip install c2cwsgiutils
 ```
 
+Or (preferred) as a base docker image:
```
Comment (sbrunner) [inline README.md]: s/docker/Docker/

File: README.md (hunk 1 above)
Comment (sbrunner) [reply README.md]: Is it tested with Python 3.5, to be sure that i't working with it?

Comment (pvalsecc) [reply README.md]: No, only tested with 3.5. Please create an issue if you want it tested with 3.5 as well.

**Pull request merged**